Makefile

```diff
@@ -6,35 +6,42 @@ NAMESPACE ?= nmstate-default
 IMAGE_REGISTRY ?= quay.io/nmstate
 IMAGE_TAG ?= latest
 PULL_POLICY ?= Always
-STATE_HANDLER_IMAGE ?= kubernetes-nmstate-state-handler
-POLICY_HANDLER_IMAGE ?= kubernetes-nmstate-configuration-policy-handler
+OPERATOR_IMAGE ?= kubernetes-nmstate
+NMSTATECTL_IMAGE ?= nmstatectl
 KUBECONFIG ?= $(shell pwd)/cluster/.kubeconfig
 
-commands = state-handler policy-handler
-func_tests_sources = $(shell find tests/ -name "*.go")
-apis_sources = $(shell find pkg/apis -name "*.go")
+manager_bin = build/_output/bin/kubernetes-nmstate
+func_tests_sources = $(wildcard tests/*.go)
+apis_sources = $(wildcard pkg/apis/**/*.go)
+pkg_sources = $(wildcard pkg/**/*.go)
+cmd_sources = $(wildcard find cmd/**/*.go)
 
-$(commands):
-	go fmt ./cmd/$@
-	go vet ./cmd/$@
-	go build -o ./bin/$@ ./cmd/$@ 
-
-build: $(commands)
+$(manager_bin): $(pkg_sources) $(cmd_sources)
+	go fmt ./cmd/manager
+	go vet ./cmd/manager
+	CGO_ENABLED=0 GOOS=linux go build \
+		-o $@ ./cmd/manager/
 
 tests/tests.test: $(func_tests_sources) $(apis_sources)
 	ginkgo build tests
 
-docker-state-handler:
-	docker build -f cmd/state-handler/Dockerfile -t $(IMAGE_REGISTRY)/$(STATE_HANDLER_IMAGE):$(IMAGE_TAG) .
+operator: $(operator_bin)
+
+docker-operator: $(operator_bin)
+	docker build -f build/Dockerfile . -t $(IMAGE_REGISTRY)/$(OPERATOR_IMAGE):$(IMAGE_TAG)
```

Comment:
There are operator-sdk build commands, we could use them. However, we have a second image to build - nmstatectl. What if we change the structure so we have build/operator and build/nmstatectl, to keep it symmetrical. I think it should not conflict with the operator-sdk generate commands.